src/ui/ResponsiveComponents/ResponsiveTabs.ts

```diff
@@ -325,14 +332,22 @@ export class ResponsiveTabs implements IResponsiveComponent {
         }
       };
 
-      tab.on('click', () => {
+      const swapOnSelect = () => {
         if (this.tabIsInDropdown(tab)) {
           let lastTabInSection = this.tabsInTabSection.pop();
           if (lastTabInSection) {
             EventsUtils.addPrefixedEvent(tab.el, 'TransitionEnd', fadeOutFadeIn);
             tab.el.style.opacity = lastTabInSection.style.opacity = '0';
           }
         }
+      };
+
+      tab.on('click', () => swapOnSelect());
```

Comment:
Should the tabs of the dropdown be accessible buttons too?

Reply:
"tabs of the dropdown" as in tab inside the dropdown ? That's what this PR does :)

Reply:
Okay, then why not use the AccessibleButton class and it's helper methods rather than having to add event listeners directly on the tab?

Reply:
Because tabs are already using "AccessibleButton", so they are already correct.

It's just that here we want to add additional logic to it on click/select (close the dropdown/swap the elements)